RealmBrowser/Views/RLMBasicTableCellView.m

```diff
@@ -32,9 +32,15 @@ - (instancetype)initWithFrame:(NSRect)frameRect
     textField.translatesAutoresizingMaskIntoConstraints = NO;
     textField.bordered = NO;
     textField.drawsBackground = NO;
-    textField.usesSingleLineMode = YES;
-    textField.lineBreakMode = NSLineBreakByTruncatingTail;
     
+    if ([textField respondsToSelector:@selector(setUsesSingleLineMode:)]) {
+        textField.usesSingleLineMode = YES;
```

Comment:
Aren't there equivalent methods on `NSTextFieldCell` that predate the methods on `NSTextField` itself? Why not use them directly on all platforms rather than conditionally using API that's not available on all OS versions we support?